src/antsibull/data/ansible-readme.rst

```diff
@@ -56,6 +56,10 @@ Issues with plugins and modules in the Ansible package should be reported
 on the individual collection's issue tracker.
 Issues with ``ansible-core`` should be reported on
 the `ansible-core issue tracker <https://github.com/ansible/ansible/issues/>`_.
+Issues with the ``ansible`` package build process or serious bugs or
+vulnerabilities in a collection that are not addressed after opening an issue
+in the collection's issue tracker should be reported on
+`ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data/issues>`_.
```

Comment:
What about linking to the README instead of directly to the issue tracker in the first place? This section is complicated enough, and I'm trying to avoid another "for more information" link.

```suggestion
`ansible-build-data's issue tracker <https://github.com/ansible-community/ansible-build-data#issue-tracker>`_.
```

Reply:
Sounds OK to me.

Reply:
@felixfontein What do you think about this latest proposal from @gotmax23?

Reply:
Sounds also fine for me!

Reply:
@gotmax23 Maybe just commit your latest suggestion and we'll do another round of reviewing. What do you think? I hope you'll get approval (and applause) all around :smirk: